docker/apply/forms-json/lawyers.json

```diff
@@ -1621,11 +1623,15 @@
       "title": "Post to lists",
       "type": "webhook",
       "outputConfiguration": {
-        "url": "http://lists:3000/ingest/lawyers"
+        "url": "http://list:3000/ingest/lawyers"
       }
     }
   ],
   "version": 2,
+  "feedback": {
+    "feedbackForm": false,
+    "url": "/smart-survey"
```

Comment:
does this need to be `https://www.smartsurvey.co.uk/s/Find-a-Professional`?

Reply:
I did that the first time but I got an error. Something about it not allowing external links. Do you think this is something that needs to be changed in the form builder?